import "google/protobuf/descriptor.proto";
extend google.protobuf.FieldOptions {
    optional bool sql_primary_key = 32382 [ default = false];
    optional bool sql_ignore      = 32383 [ default = false];
}

message DatabaseCampaignMessage {
    required int32 pilot_id      = 1 [(sql_primary_key) = true];
    required int32 variant_id    = 2 [(sql_ignore) = true];
    required int32 fspmethod_id  = 3 [(sql_ignore) = true];
    required int32 instr2        = 4 [(sql_ignore) = true];
    required int32 data_address  = 5 [(sql_ignore) = true];
}